### PR TITLE
AboutPage with styling.

### PR DESCRIPTION
**Pull request by arse0039**
Created the About Page component and started messing with styling for the photos

### CODE REVIEW

File: frontend/src/pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React, {useState,  CSSProperties } from 'react';
+import brandonImage from '../assets/BArsenault.jpg';
+
+
+
+const AboutPage: React.FC = () => {
+    const [boxShadow, setBoxShadow] = useState<string>('none');
+    const [boxPosition, setBoxPosition] = useState<string>('translateX(0) scale(1)');
+
```
Comment (davidclaphan) [inline frontend/src/pages/about.tsx]: we should add a comment that this needs to be refactored to a separate CSS file w/ the task number once we have a board set up

File: frontend/src/pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        setBoxPosition('translateX(-200px) scale(0.5)');
+    };
+
+    const handleExit = () => {
+        setBoxPosition('translateX(0px) scale(1)');
+    }
+
+    return (
+        <div style={outerStyle}>
+            <div className='aboutDavid'>
```
Comment (davidclaphan) [inline frontend/src/pages/about.tsx]: comment on this as well to note the images need to be split into 2 divs

**Review by davidclaphan — APPROVED** [review]
Updates look great, nice work on the image animations

File: frontend/src/pages/about.tsx (hunk 1 above)
Comment (arse0039) [reply frontend/src/pages/about.tsx]: I went ahead and took care of this!

**Pull request merged**